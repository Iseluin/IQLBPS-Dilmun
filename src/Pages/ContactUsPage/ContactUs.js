const ContactUs = () => {
  return (
    <div className="bg-white text-gray-100 px-8 py-12">
      <div className="max-w-screen-xl mt-24 px-8 grid gap-8 grid-cols-1 md:grid-cols-2 md:px-12 lg:px-16 xl:px-32 py-16 mx-auto text-gray-900">
        <div className="flex flex-col justify-between">
          <div>
            <h2 className="text-4xl lg:text-5xl font-bold leading-tight text-primary">
              Lets talk about everything!
            </h2>
            <div className="text-gray-600 mt-8">Give us your feedback</div>
          </div>
          <div className="mt-8 text-center">
            <img
              className="h-44 sm:h-56 md:h-72"
              src="https://alshifaeye.org/assets/images/contact-us.png"
              alt="contact us"
            />
          </div>
        </div>
        <div className="">
          <div>
            <span className="uppercase text-sm text-gray-600 font-bold">
              Full Name
            </span>
            <input
              className="w-full bg-background text-gray-900 mt- p-3 rounded-lg focus:outline-none focus:shadow-outline"
              type="text"
              placeholder=""
            />
          </div>
          <div className="mt-5">
            <span className="uppercase text-sm text-gray-600 font-bold">
              Email
            </span>
            <input
              className="w-full bg-background text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline"
              type="text"
            />
          </div>
          <div className="mt-5">
            <span className="uppercase text-sm text-gray-600 font-bold">
              Message
            </span>
            <textarea className="w-full h-40 bg-background text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline" />
          </div>
          <div className="mt-5">
            <button
              type="submit"
              className="uppercase text-sm font-bold tracking-wide bg-tertiary text-primary p-3 rounded-lg w-full focus:outline-none focus:shadow-outline hover:text-tertiary hover:bg-secondary"
            >
              Send Message
            </button>
          </div>
        </div>
      </div>
    </div>
  );
};

export default ContactUs;
