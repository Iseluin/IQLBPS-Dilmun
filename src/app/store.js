
import {
  configureStore
} from '@reduxjs/toolkit';
import userReducer from '../features/user/userSlice';
import itemReducer from '../features/slices/item'

export const store = configureStore({
  reducer: {
    user: userReducer,
    item: itemReducer
  },
});
