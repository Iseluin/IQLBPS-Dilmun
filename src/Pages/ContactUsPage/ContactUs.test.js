import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import ContactUs from './ContactUs';
import { store } from '../../app/store';

test('Contact us Page Rendered Correctly', () => {
  const tree = render(
    <Provider store={store}>
      <Router>
        <ContactUs />
      </Router>
    </Provider>
  );
  expect(tree).toMatchSnapshot();
});
