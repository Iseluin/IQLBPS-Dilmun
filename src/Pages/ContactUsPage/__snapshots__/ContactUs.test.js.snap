// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact us Page Rendered Correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bg-white text-gray-100 px-8 py-12"
      >
        <div
          class="max-w-screen-xl mt-24 px-8 grid gap-8 grid-cols-1 md:grid-cols-2 md:px-12 lg:px-16 xl:px-32 py-16 mx-auto text-gray-900"
        >
          <div
            class="flex flex-col justify-between"
          >
            <div
              class="text-center"
            >
              <h2
                class="text-4xl lg:text-5xl font-bold leading-tight text-primary "
              >
                Lets talk about everything!
              </h2>
              <div
                class="text-gray-600 mt-8 "
              >
                Give us your feedback
              </div>
            </div>
            <div
              class="mt-8 text-center self-center"
            >
              <img
                alt="contact us"
                class="h-44 sm:h-56 md:h-72"
                src="https://alshifaeye.org/assets/images/contact-us.png"
              />
            </div>
          </div>
          <form>
            <div>
              <span
                class="uppercase text-sm text-gray-600 font-bold"
              >
                Full Name
              </span>
              <input
                class="w-full bg-background text-gray-900 mt- p-3 rounded-lg focus:outline-none focus:shadow-outline"
                name="name"
                placeholder=""
                type="text"
              />
            </div>
            <div
              class="mt-5"
            >
              <span
                class="uppercase text-sm text-gray-600 font-bold"
              >
                Email
              </span>
              <input
                class="w-full bg-background text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline"
                name="email"
                type="email"
              />
            </div>
            <div
              class="mt-5"
            >
              <span
                class="uppercase text-sm text-gray-600 font-bold"
              >
                Message
              </span>
              <textarea
                class="w-full h-40 bg-background text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline"
                name="feedback"
              />
            </div>
            <div
              class="mt-5"
            >
              <button
                class="uppercase text-sm font-bold tracking-wide bg-tertiary text-primary p-3 rounded-lg w-full focus:outline-none focus:shadow-outline hover:text-tertiary hover:bg-secondary"
                type="submit"
              >
                Send Message
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bg-white text-gray-100 px-8 py-12"
    >
      <div
        class="max-w-screen-xl mt-24 px-8 grid gap-8 grid-cols-1 md:grid-cols-2 md:px-12 lg:px-16 xl:px-32 py-16 mx-auto text-gray-900"
      >
        <div
          class="flex flex-col justify-between"
        >
          <div
            class="text-center"
          >
            <h2
              class="text-4xl lg:text-5xl font-bold leading-tight text-primary "
            >
              Lets talk about everything!
            </h2>
            <div
              class="text-gray-600 mt-8 "
            >
              Give us your feedback
            </div>
          </div>
          <div
            class="mt-8 text-center self-center"
          >
            <img
              alt="contact us"
              class="h-44 sm:h-56 md:h-72"
              src="https://alshifaeye.org/assets/images/contact-us.png"
            />
          </div>
        </div>
        <form>
          <div>
            <span
              class="uppercase text-sm text-gray-600 font-bold"
            >
              Full Name
            </span>
            <input
              class="w-full bg-background text-gray-900 mt- p-3 rounded-lg focus:outline-none focus:shadow-outline"
              name="name"
              placeholder=""
              type="text"
            />
          </div>
          <div
            class="mt-5"
          >
            <span
              class="uppercase text-sm text-gray-600 font-bold"
            >
              Email
            </span>
            <input
              class="w-full bg-background text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline"
              name="email"
              type="email"
            />
          </div>
          <div
            class="mt-5"
          >
            <span
              class="uppercase text-sm text-gray-600 font-bold"
            >
              Message
            </span>
            <textarea
              class="w-full h-40 bg-background text-gray-900 mt-2 p-3 rounded-lg focus:outline-none focus:shadow-outline"
              name="feedback"
            />
          </div>
          <div
            class="mt-5"
          >
            <button
              class="uppercase text-sm font-bold tracking-wide bg-tertiary text-primary p-3 rounded-lg w-full focus:outline-none focus:shadow-outline hover:text-tertiary hover:bg-secondary"
              type="submit"
            >
              Send Message
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
